### DISCUSSION TITLE
Welcome to RTSPtoWebRTC Discussions!

### DISCUSSION THREAD

**Discussion by deepch** [deepch/RTSPtoWebRTC#154] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by allenporter** [comment]
Hi, my intro: I spend a lot of time working in Home Assistant mostly working on the [Nest](https://www.home-assistant.io/integrations/nest/) integration.  I had some experience getting Nest cameras that natively support WebRTC working, and have been contributing to `RTSPtoWeb` more than RTSPtoWebRTC by making it easier to integrate with Home Assistant through an add-on, and other misc project health contributions (e.g. docker container, release notes, etc).

Thanks for setting up the discussion.

**Comment by deepch** [comment]
Excuse me from the possible situations in Russia, I do not realize the time of the project.

I am far from politics and against war, but all this prevents me from thinking and working in this direction.

**Comment by Amirah73** [comment]
Could you tell me what is the maximum numbers of cameras that can be added to your code in Config.json file?

thanks again and again 🙏

**Comment by deepch** [comment]
10 000+

**Reply by Amirah73** [reply]
How can we determine the minimum HW requirements ?

**Reply by dbl0null** [reply]
> How can we determine the minimum HW requirements ?

Hi! Recently I've did some measurments and ended up with numbers about 2.5% of CPU usage per stream/viewer so it would be around 40 rtsp streams per CPU core (measured on some VPS Xeon)
If there's no one watching the stream right now and WebRTC adapter is not busy CPU usage gets as low as 0.8-1.1% of CPU core (up to 100 streams per CPU core)

I've used slightly modified version of RTSPtoWebRTC with audio transcoder, so it get some additional impact on numbers, around 0.5% I guess.

@deepch Can you please confirm my measurements? If so I'm wondering what HW infrastructure yu running this project on. 
(У меня получилось в районе 1% на стрим в OnDemand без зрителей и 2.5% с одним зрителем и транскодером аудио, так оно примерно?)

**Comment by sasanqc** [comment]
Hi, thanks for all your efforts, 
How can I use this module in a react application?

**Comment by Lyov-Dev** [comment]
hello . can you help me ? i have issue . I am running server with my config.json, and it catch streams from ip cameras rtsp streams, i can open it in web browser, all works correct, but i can't catch my sreams with vlc player.

**Comment by ramakrishna-psit** [comment]
For me it is working fine with local network, but when accessing from outside of network, can someone tell me what ports needs to opened in Firewall
